src/OpenConext/Profile/Value/Arp.php

```diff
@@ -40,11 +40,12 @@ public static function createWith(
         array $arp,
         AttributeDictionary $dictionary = null,
     ): self {
+
+        $arp = self::sanitizeARPData($arp);
+
         // Input validation
         foreach ($arp as $attributeInformation) {
-            if (!is_array($attributeInformation)) {
-                throw new InvalidArpDataException('The attribute information in the arp should be an array.');
-            }
+
```

Comment:
This if is moved inot the sanitizer

Reply:
In the end, you did not move it to the sanitizer right?

Reply:
@MKodde yes, 206-208

Reply:
Thanks for pointing it out. :+1: